IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,67 @@
+### IPFS All Hands Format
+
+![](https://user-images.githubusercontent.com/157609/48067634-0b5f2780-e1d1-11e8-8b67-30d7aa20896b.gif)
+
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see this issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting to act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use these weekly calls to:
+
+1. Attract skilled contributors to your project
+
+2. Motivate the community about adding support for a new feature
+
+3. Find new partners, capabilities, or challenges for your project to work on/with.
+
+4. Practice speaking in front of a captivated audience
+
+
+## The Format
+
+Every month we will have:
+
+* 15 minute talk from guest presenter (3x per month)
+* 5 minute talks lightening talks (1x per month)
+
+In total we will have a talk every Monday. The times for the talks are:
+
+| Time (PST - UTC -CET ) |
+---|
+9:00 **17:00** 18:00 |
+
+
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot ai.
```

Comment:
@pkafei the email is still incorrect. the correct domain is just @protocol.ai, therefore your email address is portia@protocol.ai